docs/source/contents/4_draft_mapping.rst

```diff
@@ -1662,6 +1662,407 @@ opaque:hostItem
+
+.. code-block:: xml
+
+    <location>
+        <physicalLocation>The University of Tennessee Libraries, Knoxville</physicalLocation>
+    </location>
+
+.. code-block:: turtle
+
+    @prefix relators: <http://id.loc.gov/vocabulary/relators> .
+
+    <https://example.org/objects/1>
+        relators:rps <http://id.loc.gov/authorities/names/n80003889> .
+
+`Example record from cDanielCartoon:1177 <https://digital.lib.utk.edu/collections/islandora/object/cDanielCartoon:1177/datastream/MODS/view>`_
+
+.. code-block:: xml
+
+    <location>
+        <physicalLocation>University of Tennessee, Knoxville. Special Collections</physicalLocation>
+        <holdingSimple>
+            <copyInformation>
+                <shelfLocator>Taxes and Economy</shelfLocator>
+            </copyInformation>
+        </holdingSimple>
+    </location>
+
+.. code-block:: turtle
+
+    @prefix relators: <http://id.loc.gov/vocabulary/relators> .
+
+    <https://example.org/objects/1>
+        relators:rps <http://id.loc.gov/authorities/names/no2014027633> .
+
+physicalLocation as string (non-UTK)
+------------------------------------
+
+Use Case
+^^^^^^^^
+Across our collections, there are also many cases where non-UTK items do not have URIs and instead use strings.
+
+Justification
+^^^^^^^^^^^^^
+Translating these to a relative URIs would require significant effort, and the value added may be trivial at this point.
+
+Xpath
+^^^^^
+
+`mods:location/mods:physicalLocation`
+
+Decision
+^^^^^^^^
+In cases that the physicalLocation are non-UTK and only a string is provided, we will only use the string literal.
+
+.. code-block:: xml
+
+    <location>
+        <physicalLocation>Blount County Public Library</physicalLocation>
+        <holdingExternal>
+            <holding xsi:schemaLocation="info:ofi/fmt:xml:xsd:iso20775 http://www.loc.gov/standards/iso20775/N130_ISOholdings_v6_1.xsd">
+                <physicalAddress>
+                    <text>City: Maryville</text>
+                    <text>County: Blount County</text>
+                    <text>State: Tennessee</text>
+                </physicalAddress>
+            </holding>
+        </holdingExternal>
+    </location>
+
+.. code-block:: turtle
+
+    @prefix relators: <http://id.loc.gov/vocabulary/relators> .
+
+    <https://example.org/objects/1>
+        relators:rps "Blount County Public Library" .
+
+physicalLocation with shelfLocator (UTK)
+----------------------------------------
+
+Use Case
+^^^^^^^^
+In many cases, some of our collection items will have shelfLocator information. This information may not currently be accurate and can found via Special Collections’ finding aids.
+
+Justification
+^^^^^^^^^^^^^
+Because our records in MODS records may not be accurate and this information is located elsewhere, and perhaps more accurate, we will drop this information when shelfLocator is used in conjunction with our repositories.
+
+Xpath
+^^^^^
+
+`mods:location/mods:shelfLocator`
```

Comment:
We made the decision a while ago to treat all xpath sections as inline code. Switch to:

```
:code:`mods:location/mods:shelfLocator`
```